pybedtools/bedtool.py

```diff
@@ -1059,24 +1076,25 @@ def introns(self, gene="gene", exon="exon"):
                     if e.start >= g.start and e.end <= g.end
                 ]
 
-                for i, exon in enumerate(exons):
+                for i, exon_instance in enumerate(exons):
```

Comment:
exon variable here was renamed to exon_instance so that it does not clash with the exon variable on the method signature.